blend4web/build.boot

```diff
@@ -0,0 +1,216 @@
+             :checksum "91034504d661782679e48a01379abbde")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/nla.js" +lib-version+)
+             :checksum "02e2adcb3ab3e4efc26f9ec0ecddd7d5")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/objects.js" +lib-version+)
+             :checksum "09c2ce7fd75ddb670f031de2476b192e")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/particles.js" +lib-version+)
+             :checksum "af58e847ffdf3ae5e5871f2b66c7efa7")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/physics.js" +lib-version+)
+             :checksum "884af1e4b71b43a5ecec5b53e8a9fc6d")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/rgb.js" +lib-version+)
+             :checksum "cf89e1d29e4e746c5fde11e1d0e5f547")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/scenes.js" +lib-version+)
+             :checksum "0a8ef3f432a801f112f9d90e34321f3e")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/sfx.js" +lib-version+)
+             :checksum "6aef021ecd1453493085f4d478e165e5")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/textures.js" +lib-version+)
+             :checksum "93b0022d9946dc4d5ba87e83c045ac41")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/time.js" +lib-version+)
+             :checksum "38687cadd01cebf654d0859ae0a7c207")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/transform.js" +lib-version+)
+             :checksum "7937ecc241322fa7e546315992149c7a")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/tsr.js" +lib-version+)
+             :checksum "05307122008c3c6b19bf5137bdae89d2")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/util.js" +lib-version+)
+             :checksum "732491e7a8c8db3cbd2fa8ca5f0f0090")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/src/ext/version.js" +lib-version+)
+             :checksum "ebb10a947f1b609fc168838a73f1357f")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/deploy/apps/common/uranium.js" +lib-version+)
+             :checksum "9B298C15DAE3D526F9652D896A088CC1")
+   (download :url (format "https://raw.githubusercontent.com/TriumphLLC/Blend4Web/%s/deploy/apps/common/uranium.js.mem" +lib-version+)
+             :checksum "3FF638D920819952240A1FDC28E6FD16")
+   (sift :move {;;These are all the externs
+                #"^b4w.js"
+                "cljsjs/blend4web/common/b4w.ext.js"
```

Comment:
Oh sweet - I didn't even know about that `:decompress` flag :P  While I like the idea of a shorter `build.boot`, the zip is 1.4 gigs big.  From a Canadian internet perspective, this means a long download time.  But that's my bias.